k8s-athenz-identity/templates/test-app/test-app.yaml

```
@@ -0,0 +1,209 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: athenz-identity-test-app
+  name: athenz-identity-test-app
+  namespace: {{ .Values.test_app.namespace }}
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: athenz-identity-test-app
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: athenz-identity-test-app
+      name: athenz-identity-test-app
+    spec:
+      containers:
+      - args:
+        - --mode=refresh
+        - --provider-service {{ .Values.test_app.provider_service }}
```

Sorry that I haven't tried it using helm, but I think it should be 
```suggestion
        - --provider-service={{ .Values.test_app.provider_service }}
```

---

Ah yes, this should have the equal sign. Updated.